anyway/views/news_flash/api.py

```diff
@@ -10,19 +10,23 @@
 from anyway.base import user_optional
 from anyway.models import NewsFlash
 
+DEFAULT_OFFSET = 0
+DEFAULT_LIMIT = 100
```

Comment:
I think it is better to qualify the names of these constants so it will be clear they relate to request parameters